### DISCUSSION TITLE
Two different repos' release merged

### DISCUSSION THREAD

**Discussion by polybiusproxy** [community/community#66652] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

A `llama.cpp` (AI model) release and a `PCSX2` (PS2 emulator) release were merged together, somehow.
They are two unrelated projects.
![image](https://github.com/community/community/assets/47796739/5d3fa02a-efd5-49b1-aa59-4c6e14f695fa)

**Comment by StripedMonkey** [comment]
In the included screenshot `lapce/lapce` and `Cataclysm-DDA experimental build ...` are two independent, unrelated projects. In the screenshot it looks like it's the *exact same project*. This is horrible UX and leads to confusion as to what lapce is doing with release notes from Cataclysm-DDA.
![image](https://github.com/community/community/assets/8009284/410423eb-db32-4ed1-b082-9257838fbdcc)
